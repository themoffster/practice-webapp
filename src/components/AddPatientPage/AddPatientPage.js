import React from 'react'

const AddPatientPage = () => {
  return (
    <>
      <span>foo</span>
    </>
  )
}

AddPatientPage.propTypes = {}

export default AddPatientPage
