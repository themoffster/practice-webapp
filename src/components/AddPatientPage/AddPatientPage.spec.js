import { render } from '@testing-library/react'
import AddPatientPage from './AddPatientPage'
import React from 'react'

const getWrapper = () => render(<AddPatientPage />)

describe('AddPatientPage', () => {
  it('matches the snapshot', () => {
    const { getFragment } = getWrapper()
    expect(getFragment).toMatchSnapshot()
  })
})
