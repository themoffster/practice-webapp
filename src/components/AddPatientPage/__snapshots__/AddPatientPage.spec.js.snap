// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddPatientPage matches the snapshot 1`] = `undefined`;
